util/gen_log_guide.ts

```diff
@@ -0,0 +1,169 @@
+import path from 'path';
+
+import markdownMagic from 'markdown-magic';
+
+import { isLang, languages, NonEnLang } from '../resources/languages';
+import logDefinitions, { LogDefinitionTypes } from '../resources/netlog_defs';
+import NetRegexes from '../resources/netregexes';
+import Regexes from '../resources/regexes';
+import LogRepository from '../ui/raidboss/emulator/data/network_log_converter/LogRepository';
+import ParseLine from '../ui/raidboss/emulator/data/network_log_converter/ParseLine';
+
+type LangObject<T> = { en: T } & { [lang in NonEnLang]?: T };
+
+const curPath = path.resolve();
+
+// Exclude these types since they're not relevant
+type ExcludedLineDocs = 'None';
+type LineDocTypes = Exclude<LogDefinitionTypes, ExcludedLineDocs>;
+
+type LineDocType = {
+  // We can generate `network` type automatically for everything but regex
+  regexes: {
+    network: string;
+    logLine: string;
+  };
+  examples: LangObject<readonly string[]>;
+};
+
+type LineDocs = {
+  [type in LineDocTypes]: LineDocType;
+};
+
+// @TODO: Remove Partial once everything is mapped
+const lineDocs: Partial<LineDocs> = {
+  GameLog: {
+    regexes: {
+      network: NetRegexes.gameLog({ capture: true }).source,
+      logLine: Regexes.gameLog({ capture: true }).source,
+    },
+    examples: {
+      en: [
+        '00|2021-04-26T14:12:30.0000000-04:00|0839||You change to warrior.|d8c450105ea12854e26eb687579564df',
```

Comment:
It might be worth having a log line that includes a name too, like Nael rp dialog or something.

Reply:
I don't have any network log versions of Nael quotes handy. I do have this Tsukuyomi quote I have included on the latest commit.

```
00|2021-07-05T18:01:21.0000000-04:00|0044|Tsukuyomi|Oh...it's going to be a long night.|1a81d186fd4d19255f2e01a1694c7607
```